components/view_image/index.ts

```diff
@@ -6,19 +6,21 @@ import {connect} from 'react-redux';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 import {getPost} from 'mattermost-redux/selectors/entities/posts';
 
-import {canDownloadFiles} from 'utils/file_utils.jsx';
+import {GlobalState} from 'mattermost-redux/types/store';
```

Comment:
i quickly tried your code and seems that you need a different definition for globalstate, one that contemplates plugins:
```suggestion
import {GlobalState} from 'types/store';
```